charts/crowdsec/templates/lapi-deployment.yaml

```diff
@@ -72,12 +80,25 @@ spec:
         env:
           - name: MB_DB_FILE
             value: /metabase-data/metabase.db
-      {{ end }}
+      {{- end }}
       terminationGracePeriodSeconds: 30
-      {{ if .Values.lapi.dashboard.enabled }}
+      {{- if or (.Values.lapi.persistentVolume.data.enabled) (.Values.lapi.persistentVolume.config.enabled) (.Values.lapi.dashboard.enabled) }}
       volumes:
+      {{- if .Values.lapi.dashboard.enabled }}
```

Comment:
This check isn't needed, as we already check the same variable to deploy all the container resource.

Reply:
Sorry I don't get this, for me it looks right. We are not inside a container in that line. We need to define the shared-data volume in case the dashboard is enabled. But on this level we can not guarantee that the dashboard is enabled

Reply:
Sorry I was not clear about it. The thing is that check `{{- if .Values.lapi.dashboard.enabled }}` is already in line 71 https://github.com/crowdsecurity/helm-charts/blob/b24bd8dd1573e9496073b54e0e97ee3d9a74b0ec/charts/crowdsec/templates/lapi-deployment.yaml#L71 

So yes it's still correct but just redundant.

Reply:
Sorry I was not clear about it. The thing is that check `{{- if .Values.lapi.dashboard.enabled }}` is already in line 71 https://github.com/crowdsecurity/helm-charts/blob/b24bd8dd1573e9496073b54e0e97ee3d9a74b0ec/charts/crowdsec/templates/lapi-deployment.yaml#L71 

So yes it's still correct but just redundant.

Reply:
I think this is also resolved or? We need the check to decide if we want to deploy the dashboard or not. There is no other check before beside of the one for the init container

Reply:
Sorry for the delay. I just tested the helm chart too. Seems good :+1: